textfinding_indikator4.py

```diff
@@ -0,0 +1,154 @@
+# this code is for TEXT FINDING - INDIKATOR 4
+# input is txt file from preprocess dokumen lama, nama instansi & judul from dokumen lama & baru
+
+import re
+import preprocess_dokbaru as dokbaru
+import preprocess_doklama as doklama
+from utility import *
+
+
+def txtreader(filename, lv, keyword):
+    # func to search keyword in txt file
+
+    # open txt file
+    file = open(f'cleaned_{filename}.txt', 'r')
+
+    idx = 0
+    result = []
+
+    # read line by line from txt
+    for line in file:
+
+        if (lv == 1):
+            reg = f'(?:(pembangunan)(.*?)(aplikasi)\s+(spbe)?)'
+            if (re.search(reg, line, re.IGNORECASE)):
+                result.append([idx, line])
+
+        elif (lv == 2):
+            for key in list(keyword):  # create a duplicate list
```

Comment:
Same as [#3 (comment)](https://github.com/Tugas-Akhir-SPBE-IF-2019/ta-spbe-textfinding/pull/3#discussion_r1140977512)

Reply:
already using duplicate list using list(keyword) and included validation if the key still exists before removing